lib/shared/addon/utils/azure-choices.js

```diff
@@ -40,6 +40,14 @@ export let regions = {
       'name':        'westeurope',
       'displayName': 'West Europe',
     },
+    {
+      'name':        'germanynorth',
+      'displayName': 'Germany North',
+    },
+    {
+      'name':        'germanywescentral',
```

Comment:
Hi Ádám,
I'm not sure, but I guess you have a typo here, haven't you? I assume the region is name "germanywestcentral", so your commit is missing a "t".

Cheers
Turgut

Reply:
Whops, nice catch!